06_deploy_bootstrap_vm.sh

```diff
@@ -6,63 +6,25 @@ source ocp_install_env.sh
 source common.sh
 source utils.sh
 
-# FIXME this is configuring for the libvirt backend which is dev-only ref
-# https://github.com/openshift/installer/blob/master/docs/dev/libvirt-howto.md
-# We may need some additional steps from that doc in 02* and also to make the
-# qemu endpoint configurable?
 if [ ! -d ocp ]; then
     mkdir -p ocp
-    cat > ocp/install-config.yaml << EOF
+    cat > ocp/baremtal-install-config.yaml << EOF
```

Comment:
I don't think we really need to change the name do we?  E.g install-config.yaml is unrelated to the platform, so just specifying the platform below should be enough?

Reply:
Yeah, kni-install is still using ocp/install-config.yaml - it's just that I'm running kni-install twice, and it deletes install-config.yaml when it runs

Reply:
Yeah that's kind of an annoying "feature" of the installer ;)

One workaround is we run create ignition-configs, save the resulting configs, then run create cluster (with the pre-generated assets)

I guess this can go away anyway when we deploy the masters via the bootstrap VM

Reply:
Right, I was thinking we show that we'll just do "create cluster" and we will work ourselves away from having to generate ignition configs at all

Reply:
I was thinking some more about this, maybe we should just deploy the masters via terraform (even if that means writing a new provider)?

I'm not certain if long term openshift-install plans to stick with terraform, but it's used now, and we could more easily fit into the current "create cluster" flow if we did the same?

Not sure if we can add ironic support to the existing openstack provider or if it would be better to have a metalkube specific one with the minimal logic to deploy nodes.

Reply:
Thoughts about provisioning masters captured here - https://github.com/metalkube/kni-installer/blob/master/docs/dev/baremetal.md